app/assets/javascripts/locomotive/not_logged_in.js.coffee

```diff
@@ -9,3 +9,20 @@ window.Locomotive = {}
 $ ->
   _.each window.flash_messages, (couple) ->
       Locomotive.notify couple[1], couple[0]
+
+  # JavaScript to render the form wrapper vertically in the middle.
```

Comment:
I still think flex is not supported widely enough. IE 8, 9 and 10 is not supported. neither are older android browsers: http://caniuse.com/#search=flex (also older safari versions do not support it).

It's your call :)